client/src/components/Applications/AppForm/AppFormUtils.js

```diff
@@ -82,7 +90,11 @@ export const createMaxRunTimeRegex = (maxRunTime) => {
  * @param {Object} queue
  * @returns {Yup.number()} min/max validation of node count
  */
-export const getNodeCountValidation = (queue) => {
+export const getNodeCountValidation = (queue, app) => {
+  if (isJobTypeFork(app)) {
+    return Yup.number().integer().min(1).max(1);
```

Comment:
Updated code based on Sal's comment, so this is not relevant anymore. 
for future reference, 
`Yup.number().integer().min(1).max(1)` returns Yup validation schema and follows Yup semantics.
return 1 will fail at runtime because of above.